applications/MeshingApplication/tests/test_MeshingApplication_mpi.py

```diff
@@ -0,0 +1,48 @@
+# import Kratos
+import KratosMultiphysics
+
+# Import Kratos "wrapper" for unittests
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+try:
+    import KratosMultiphysics.mpi as KratosMPI
+    import KratosMultiphysics.MetisApplication as MetisApplication
+    import KratosMultiphysics.TrilinosApplication as TrilinosApplication
+except ImportError:
+    raise Exception("KratosMPI could not be imported!")
+
+if KratosMultiphysics.ParallelEnvironment.GetDefaultSize() != 2:
+    raise Exception("The MPI tests currently support only being run with 2 processors!")
```

Comment:
```suggestion
```

this should not be here any more, please check `test_CoSimulationApplication_mpi.py`, this is the most up to date one